Microsoft/WebTemplateStudio
# angular README fix

This PR is to fix one of the feedback given by Angular expert, here is the source:
https://github.com/Tanya0609/angular/pull/1/files

How to test this PR:
1. build your extension (no need to package templates anymore)
2. generate an angular templates 
3. check the README file in the generated templates
